src/main/java/com/sap/dsc/aas/lib/config/pojo/AbstractConfigFromAttribute.java

```diff
@@ -23,7 +23,7 @@ public String getXPath() {
     @JsonProperty("from_xpath")
     public void setXPath(String xPath) {
         if (this.xPath != null) {
-            throw new AlreadyDefinedException("from_attributeName");
+            throw new AlreadyDefinedException("from_xpath_builder");
```

Comment:
seems to be superfluous as from_xpath_builder is not available anymore

Reply:
no longer needed but still remains